lib/generate-destination-data.js

```diff
@@ -14,6 +16,18 @@ export default function generateDestinationData ({
   transitiveNetwork,
   waitTimes,
   walkTimes
+}: {
+  from: Point,
+  inVehicleTravelTimes: Uint8Array,
+  log: Function,
```

Comment:
[The Function type is unsafe and should be avoided](https://flow.org/en/docs/types/functions/#toc-function-type).